src/main/java/pages/BottomMenuPage.java

```diff
@@ -34,32 +34,35 @@ public BottomMenuPage() {
     }
 
     public boolean isDisplayed() {
-        if (AndroidDriverSingletone.getSingletoneInstance().getDriverInstance().findElements(By.id(BOTTOM_MENU_ID)).size() == 0)
-            return false;
-        return bottomMenu.isDisplayed();
+        return (AndroidDriverSingletone.getSingletoneInstance().getDriverInstance().findElements(By.id(BOTTOM_MENU_ID)).size() != 0) && bottomMenu.isDisplayed();
```

Comment:
Please implement new methods in BasePage:
protected List<WebElement> findElementsById(String elementId)
protected WebElement findElementById(String elementId)
protected List<WebElement> findElementsByXpath(String elementXpath)
protected WebElement findElementByXpath(String elementXpath)

- [ ] Implement methods